README.md

```diff
@@ -1 +1,10 @@
-# android-mail
\ No newline at end of file
+# Infomaniak kMail
+
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
+     alt="Get it on F-Droid"
+     height="80">](https://f-droid.org/packages/com.infomaniak.mail/)
+[<img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png"
+     alt="Get it on Google Play"
+     height="80">](https://play.google.com/store/apps/details?id=com.infomaniak.mail)
```

Comment:
We prefer to put the google play in first place, because the f-droid version is devoid of push notification
There should be a small text explaining this in the readme

Reply:
Done